sass/components/_apps-modal.scss

```diff
@@ -0,0 +1,7 @@
+#appsModal {
+    .modal-footer {
+        .btn {
+            margin-bottom: 5px;
```

Comment:
This change is specifically for the last screenshot in the description, when we have an overflow amount of buttons.